protos/core/Btfs.proto

```diff
@@ -0,0 +1,230 @@
+syntax = "proto3";
+
+
+
+package btfs;
```

Comment:
Per discussion with @wxue and @jin-tron , we will put all .proto files under `protos/` directly. This file should be renamed guard.proto

The package name should be shared as well.

Reply:
Done the change